auth-api/src/auth_api/models/task.py

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 """This model manages a Task item in the Auth Service."""
 
-from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, text
-from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, text, func
+from sqlalchemy.dialects.postgresql import ARRAY  # noqa: I005
```

Comment:
What is #noqa: I005 for?

Reply:
to tell the linter to ignore that line since it was saying `missing import` even though the import was fine